src/test/java/api/support/http/CqlQuery.java

```diff
@@ -13,6 +13,10 @@ public static CqlQuery queryFromTemplate(String queryTemplate,
     return new CqlQuery(String.format(queryTemplate, parameters));
   }
 
+  public static CqlQuery exactMatch(String indexName, String value) {
+    return queryFromTemplate("%s==\"%s\"", indexName, value);
```

Comment:
Is the caller required to mask `value`? If yes the javadoc should say so. Otherwise this method should mask `value` to prevent CQL injection.
`value.replace("\\", "\\\\").replace("\"", "\\\"");`

Reply:
Hi @julianladisch, this class is used for tests only, should I add the logic anyway?

Do you know if RMB takes care about SQL injections? We have pretty same logic in production code and I don't think that we have such a logic there.

Reply:
Personally, I don't think we should try to mitigate injection in test code

Reply:
Please feel free to continue this conversation. I am merging this pull request so that I can release the bug fix version for this change.